internal/projects/creator.go

```diff
@@ -105,6 +108,15 @@ func (p *projectCreator) ProvisionSelfEnrolledProject(
 		return nil, fmt.Errorf("failed to create default project: %v", err)
 	}
 
+	// Retrieve the membership-to-feature mapping from the configuration
+	projectFeatures := p.featuresCfg.GetFeaturesForMemberships(ctx)
+	if err := qtx.CreateEntitlements(ctx, db.CreateEntitlementsParams{
+		Column1: projectFeatures,
+		Column2: project.ID,
```

Comment:
Can we rename these parameters from `Column1` and `Column2`?  😁

Reply:
Absolutely 😄